--- a/debian/rules	2014-09-16 14:06:41.000000000 +0000
+++ b/debian/rules	2014-09-16 18:41:52.285643810 +0000
@@ -49,6 +49,7 @@
 		--with-mail \
 		--with-mail_ssl_module \
 		--with-file-aio \
+                --add-module=/tmp/mods/nginx-auth-ldap \
 		$(WITH_SPDY) \
 		--with-cc-opt="$(CFLAGS)" \
 		--with-ld-opt="$(LDFLAGS)" \
@@ -87,6 +88,7 @@
 		--with-mail \
 		--with-mail_ssl_module \
 		--with-file-aio \
+                --add-module=/tmp/mods/nginx-auth-ldap \
 		$(WITH_SPDY) \
 		--with-cc-opt="$(CFLAGS)" \
 		--with-ld-opt="$(LDFLAGS)" \
